codecparsers/h265Parser.cpp

```diff
@@ -262,29 +291,29 @@ bool Parser::profileTierLevel(ProfileTierLevel* ptl, NalReader& nr,
         || ptl->general_profile_compatibility_flag[4]
         || ptl->general_profile_compatibility_flag[5])
         //The number of bits in this syntax structure is not affected by this condition
-        ptl->general_inbld_flag = nr.read(1);
+        READ(nr, ptl->general_inbld_flag);
     else
         nr.skip(1); // general_reserved_zero_bit
-    ptl->general_level_idc = nr.read(8);
+    READ_BITS(nr, ptl->general_level_idc, 8);
```

Comment:
if it read uint8_t or uint16_t, uint32_t or bool you should use READ()
this becuase we use template and template specialization trick in c++ like this

> template <class T>
> bool BitReader::readT(T& v)
> {
>     return readT(v, sizeof(v) << 3);
> }
> 
> bool BitReader::readT(bool& v)
> {
>     return readT(v, 1);
> }
>